CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## Developer Changelog
 
+### May 13, 2022
+* added new migrations and seeds for picklists database table
+
```

Comment:
Consider using "backend" prefix to show where you made changes in this version.

Reply:
Good catch, thanks.